server/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: ..
       dockerfile: ./server/src/Dockerfile
-    command: uvicorn server.src.app.main:app --reload --workers 1 --host 0.0.0.0 --port 8000
+    command: uvicorn server.src.app.main:app --reload --workers 1 --host 0.0.0.0 --port 80
```

Comment:
Just for reference, why 80 instead of 8000 ?

Reply:
no specific reason, it wasn't on purpose. juste I didn't put back the right port after testing stuff. I will move everything back to 8000